expression-test/expression_test_parser.cpp

```diff
@@ -439,33 +439,7 @@ optional<Value> toValue(const expression::Value& exprValue) {
             std::vector<Value> color { double(c.r), double(c.g), double(c.b), double(c.a) };
             return {Value{std::move(color)}};
```

Comment:
color.toObject ?

Reply:
in the follow up pr :-)  the representation there is different